python/CMakeLists.txt

```diff
@@ -63,17 +63,22 @@ declare_mlir_python_extension(AirPythonExtensions.MLIR
     LLVMSupport
 )
 
-declare_mlir_python_extension(AirPythonExtensions.AIRRt
-  MODULE_NAME _airRt
-  ADD_TO_PARENT AirPythonExtensions
-  ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}
-  SOURCES
-    AirHostModule.cpp
-  EMBED_CAPI_LINK_LIBS
-    AIRCAPI
-  PRIVATE_LINK_LIBS
-    LLVMSupport
-)
+# Only building this if we are building the runtime, as it requires
+# the headers from the runtime
+if(AIR_RUNTIME_TEST_TARGET_VAL)
```

Comment:
It's going to work the same either way but `AIR_RUNTIME_TARGETS` is what determines if the runtime is built.

Reply:
Thanks for pointing that out. Addressed in [24ea273](https://github.com/Xilinx/mlir-air/pull/380/commits/24ea2732edf24120d953affbbe585025173f59e4).